src/main/java/org/primefaces/component/datatable/DataTable.java

```diff
@@ -294,18 +297,6 @@ public void processValidators(FacesContext context) {
         }
     }
 
-    @Override
-    public void processUpdates(FacesContext context) {
-        super.processUpdates(context);
-
-        ValueExpression selectionVE = getValueExpression(PropertyKeys.selection.toString());
-
-        if (selectionVE != null) {
-            selectionVE.setValue(context.getELContext(), getLocalSelection());
-            setSelection(null);
-        }
-    }
-
```

Comment:
@tandraschko @melloware I remember you guys faced an issue with this code recently, but I can't recall what it was... Can you remind me what was the issue number?

Reply:
can only remember that the selection attr was broken in ui:repeat.

Reply:
Got it #3812